.github/scripts/config.sh

```diff
@@ -92,6 +92,8 @@ FFMPEG_BRANCH=${FFMPEG_BRANCH:-release/6.1}
 FFMPEG_PATH=${FFMPEG_PATH:-~/ffmpeg}
 FFMPEG_TESTS_PATH=${FFMPEG_TESTS_PATH:-~/ffmpeg-tests}
 
+CYGWIN_TESTSUITE=${CYGWIN_TESTSUITE:-1} #wheather to Buil & Run cygwin Internal testsuite
```

Comment:
Please revert this change and the change in `.github/scripts/build.sh`. The tests script will be executed from GitHub Actions workflow or manually from WSL shell.